examples/server/server.cpp

```diff
@@ -2295,7 +2303,15 @@ struct server_context {
         res->oaicompat         = slot.params.oaicompat;
         res->oaicompat_model   = slot.params.oaicompat_model;
         res->oaicompat_cmpl_id = slot.params.oaicompat_cmpl_id;
-
+        if (slot.params.chat_parser) {
+            res->oaicompat_chat_msg = slot.params.chat_parser(slot.generated_text);
+        } else {
+            res->oaicompat_chat_msg = {
+                /* .role = */ "assistant",
+                /* .content = */ slot.generated_text,
```

Comment:
If we're sure that we don't use `slot.generated_text` afterwards, probably `std::move(slot.generated_text)` to avoid string copy

Reply:
Done